src/exchange_adapters/gemini.ts

```diff
@@ -0,0 +1,76 @@
+import { Exchange } from '../utils'
+import { BaseExchangeAdapter, ExchangeDataType, Ticker, Trade } from './base'
+
+export class GeminiAdapter extends BaseExchangeAdapter {
+  baseApiUrl = 'https://api.gemini.com/v1/'
+
+  readonly _exchangeName = Exchange.GEMINI
+  // Amazon cert
+  readonly _certFingerprint256 =
+    'F5:5F:9F:FC:B8:3C:73:45:32:61:60:1C:7E:04:4D:B1:5A:0F:03:4B:93:C0:58:30:F2:86:35:EF:88:9C:F6:70'
+
+  async fetchTicker(): Promise<Ticker> {
+    return this.parseTicker(
+      await this.fetchFromApi(ExchangeDataType.TICKER, `pubticker/${this.pairSymbol}`)
+    )
+  }
+
+  async fetchTrades(): Promise<Trade[]> {
+    /**
+     * Trades are cool, but empty arrays are cooler.
+     *                          @bogdan, 01/2023
+     */
+    return []
+  }
+
+  protected generatePairSymbol(): string {
+    return `${BaseExchangeAdapter.standardTokenSymbolMap.get(
+      this.config.baseCurrency
+    )}${BaseExchangeAdapter.standardTokenSymbolMap.get(
+      this.config.quoteCurrency
+    )}`.toLocaleLowerCase()
```

Comment:
nit: how about separating these in variables to make it a bit more readable?

Reply:
✅